modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats/index.css

```diff
@@ -1,32 +1,49 @@
-.congrats {
-	text-align: center;
-}
-
 .congrats h1 {
 	color: var(--color-action-primary-default);
-	font-size: 43px;
-	font-weight: 800;
+	font-size: 43px;	
 	height: 48px;
 	margin-bottom: 34px;
 }
 
 .congrats h2 {
 	color: var(--color-neutral-9);
 	font-size: 21px;
-	font-weight: 800;
 	line-height: 32px;
 }
 
 .congrats h3 {
 	color: var(--color-neutral-9);
 	font-size: 21px;
-	font-weight: 500;
 	line-height: 32px;
 }
 
 .congrats h4 {
 	color: var(--color-neutral-9);
 	font-size: 16px;
-	font-weight: 500;
 	line-height: 24px;
+}
+
+.you-did-it{
```

Comment:
```suggestion
.you-did-it {
```

Reply:
What you guys think @samuelleolopes , @silasbsi ??

Reply:
make sense.